src/Umbraco.Cms.ManagementApi/Services/ExamineManagerService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Examine;
+using Umbraco.Cms.Infrastructure.Examine;
+
+namespace Umbraco.Cms.ManagementApi.Services;
+
+public class ExamineManagerService : IExamineManagerService
```

Comment:
I think the only justification for this class is the `TryFindSearcher` method. The rest of the methods are just renaming proxied methods on `IExamineManager` or `IIndexRebuilder`.

I would consider reshaping this class to only be used for finding a searcher, and leave the rest of the implementations to be used as-is from their respective services.

I'll add a few more comments to this file, they can be disregarded if the class is reshaped 😄